index.html

```diff
@@ -131,6 +131,39 @@
 								</v-btn>
 							</v-card>
 
+                     <v-menu offset-y>
+                        <template v-slot:activator="{ on, attrs }">
+                           <v-btn
+                              class="ml-2"
+                              v-bind="attrs"
+                              v-on="on"
```

Comment:
should have same border radius as all the other buttons

Reply:
Button was transparent so whatever and will be placed somewhere else